File: problemas/strings/1871/1871.py
def main():
    while True:
        M, N = map(int, input().split(' '))
        soma = M + N
        if soma == 0:
            break
        string = str(soma)

        print(f"{string}" if '0' not in string else string.replace('0', ''))

if __name__ == "__main__":
    main()

File: problemas/strings/1871/1871.ts
process.stdin.resume();
process.stdin.setEncoding('utf8');

let input: string = "";
let lineCount: number = 0;

process.stdin.on("data", (chunk: string) => {
    input += chunk;
});

process.stdin.on("end", () => {
    let lines = input.split('\n');

    while (lines.length) {
        let [M, N] = lines.shift()!.trim().split(' ').map((x) => parseInt(x));

        if (!M && !N) {
            break;
        }

        let soma = (M + N).toString();
        let resposta = soma.split('0').join('');

        console.log(resposta);
    }

});
